dev/integration_tests/android_engine_test/lib/hcpp/platform_view_main.dart

```diff
@@ -21,25 +21,53 @@ void main() async {
   runApp(const MainApp());
 }
 
-final class MainApp extends StatelessWidget {
+// This should appear as the yellow line over a blue box. The
+// green box should not be visible unless the platform view has not loaded yet or
+// has been removed from the scene.
+final class MainApp extends StatefulWidget {
```

Comment:
As a note, it might make sense to add this style of overlay test for all the PV backends (instead of HC++ specific), and then instead revert the box I added in https://github.com/flutter/flutter/pull/162903 (I like your implementation more).

It doesn't have to be in this PR, but do it as an immediate follow-up.

Reply:
Done